### DISCUSSION TITLE
somebody can help me please ?

### DISCUSSION THREAD

**Discussion by johnwayne7** [vercel/community#3881] [Account Issue]
### Question

I need to have a new count inside vercel, I give my email, and now I'm blocked. Can you help me please ?

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
